src/main/java/org/fundacionjala/core/selenium/browsers/config/BrowserProperties.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+      "name": "Mozilla Firefox",
+      "version": "73.0.1",
+      "implicitWaitingSeconds": "30",
+      "explicitWaitingSeconds": "40",
+      "sleepingTimeMills": "500"
+    },
+    {
+      "name": "Microsoft Edge",
+      "version": "87.0.664.66",
+      "implicitWaitingSeconds": "40",
+      "explicitWaitingSeconds": "50",
+      "sleepingTimeMills": "500"
+    },
+    {
+      "name": "Google Chrome",
+      "version": "87.0.4280.88",
+      "implicitWaitingSeconds": "30",
+      "explicitWaitingSeconds": "40",
+      "sleepingTimeMills": "500"
+    }
+]
```

Comment:
I think you need add 
"flags" or "arguments" in all collections 
and inside should be arguments for execute the browser, for example the resolution or run in background 

flags: [
       "--headless",
      "--disable-gpu",
      "--no-sandbox",
      "--window-size=1920,1200"
]